service/src/main/java/uk/gov/hmcts/reform/fpl/utils/JudgeAndLegalAdvisorHelper.java

```diff
@@ -25,6 +26,10 @@ public static String formatJudgeTitleAndName(JudgeAndLegalAdvisor judgeAndLegalA
     }
 
     private static String mapJudgeOrAdvisor(JudgeAndLegalAdvisor judgeAndLegalAdvisor) {
+        if (isBlank(judgeAndLegalAdvisor.getJudgeLastName()) && isBlank(judgeAndLegalAdvisor.getJudgeFullName())) {
```

Comment:
I could not find it in the story, is that something business wanted? It hides judge title if name is missing, is that alright?

If this is OK please update tests.

Reply:
It may not be captured in story but it is what business wants (it may be noted in 1004, these stories are linked, it's a bit messy) - there is a situation where I draft SDO, then I redraft it later and remove a judge title. In this case the document will show the judge title with no name I.e "Her Honour Judge", but intended behaviour is that it should give the empty placeholder. If we have judge title we must have judge name. In following 1004 story, we then deal with magistrate situation. If we have magistrate title, then we do not need name and will display something different.

regardless yes I think this needs updated test, I will add 👍

Reply:
test added, please approve changes if happy